net/blockwise/blockwise.go

```diff
@@ -523,10 +523,8 @@ func isObserveResponse(msg *pool.Message) bool {
 	if err != nil {
 		return false
 	}
-	if msg.Code() == codes.Content {
-		return true
-	}
-	return false
+	// TODO: codes.Valid?
```

Comment:
yes also valid is fine

Reply:
Also we need to check it here:
https://github.com/plgd-dev/go-coap/blob/c1d18474cd9046c8a466465b2f5bbbd55309f410/net/blockwise/blockwise.go#L750